scipy/stats/tests/test_distributions.py

```diff
@@ -3236,6 +3236,48 @@ def test_logsf(self):
         y = stats.rayleigh.logsf(50)
         assert_allclose(y, -1250)
 
+    @pytest.mark.parametrize("floc", [-1, 0, 1, 2, 3])
+    @pytest.mark.parametrize("fscale", [.5, 1, 2, 3, 4])
+    def test_fit(self, floc, fscale):
+        data = stats.rayleigh.rvs(size=250, loc=floc, scale=fscale)
+
+        def scale_mle(data, floc):
+            return (np.sum((data - floc) ** 2) / (2 * len(data))) ** .5
+
+        scale_expect = scale_mle(data, floc)
+        # when `floc` is provided, direct MLE equation is used
+        loc, scale = stats.rayleigh.fit(data, floc=floc)
+        assert_equal(loc, floc)
+        assert_equal(scale, scale_expect)
+        # when `fscale` is fixed, standard numerical optimization is used.
+        loc, scale = stats.rayleigh.fit(data, fscale=.6)
+        assert_equal(scale, .6)
```

Comment:
What about `loc`?

Reply:
Do you mean how is `loc` calculated, or whether there is a test regarding `loc`?

Reply:
comment adjusted to indicate that `loc` is calculated by the superclass.